frontend/src/lib/components/common/button/Button.svelte

```diff
@@ -63,8 +63,10 @@
 	}
 
 	function onClick(event: MouseEvent) {
-		dispatch('click', event)
-		if (href) goto(href)
```

Comment:
Why can you remove the goto ?

Reply:
Adding the goto is the fix that I to do to avoid buttons breaking up navigation by going to a new page instead of using svelte internal navigation. Propagating href seems to not work as you expect it to.